src/models/People.js

```diff
@@ -39,11 +39,9 @@ static async displayPeopleProfessional() { // returns array of Timesheet Table
 }*/
   
   static async login(email, password) {
-    try {
-      
       const db = new DB();
       const sql = `SELECT id, salt, first_name, last_name, force_password_change FROM people 
-        WHERE email=:email AND encrypted_password = sha2(CONCAT(:password, salt), 256) AND login_enabled = 1 LIMIT 1;`; 
+        WHERE email=:email AND encrypted_password = sha2(CONCAT(:password, salt), 256) AND login_enabled = 1 AND approved = 1 LIMIT 1;`; 
```

Comment:
We need a migration folder here in `klaudsol-commons` as well. Kindly put it in `/src/db/migrations/20220414171000-sample.json`